catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -33,7 +33,7 @@
 	// Reflection in testcases needed
 	// http://stackoverflow.com/questions/1615163/modifying-final-fields-in-java?answertab=votes#tab-top
 
-	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = Float.valueOf(0.993f);
+	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = Float.valueOf(0.994f);
```

Comment:
Well, I think this is a point of discussion since it can also be raised together with future language version changes, when it really is necessary.
Whatever, I've added that line now ;)